src/bionics.cpp

```diff
@@ -723,6 +731,33 @@ void player::process_bionic( int b )
     } else if (bio.id == "bio_hydraulics") {
         // Sound of hissing hydraulic muscle! (not quite as loud as a car horn)
         sounds::sound( pos(), 19, _("HISISSS!"));
+    } else if( bio.id == "bio_cable" ) {
+        const std::vector<item*> cables = items_with( []( const item &it ) {
+            return it.active && it.has_flag( "CABLE_SPOOL" );
```

Comment:
Some minor duplication here. Please consider assigning the lambda to a constant.